python/ctsm/crop_calendars/regrid_ggcmi_shdates.py

```diff
@@ -32,49 +42,95 @@ def define_arguments(parser):
     parser.add_argument(
         "-rt",
         "--regrid-template-file",
-        help="Template file to be used in regridding of inputs.",
+        help="Template netCDF file to be used in regridding of inputs. This can be a CLM output file (i.e., something with 1-d lat and lon variables) or a CLM surface dataset (i.e., something with 2-d LATIXY and LONGXY variables).",
         type=str,
         required=True,
     )
     return parser
 
 
 def main(
-    regrid_resolution, regrid_template_file_in, regrid_input_directory, regrid_output_directory
+    regrid_resolution,
+    regrid_template_file_in,
+    regrid_input_directory,
+    regrid_output_directory,
+    extension,
+    crop_list,
 ):
+    print(f"Regridding GGCMI crop calendars to {regrid_resolution}:")
 
-    # Ensure we can call necessary shell scripts
-    for cmd in ["ncks", "ncrename", "ncpdq", "cdo"]:
+    # Ensure we can call necessary shell script(s)
+    for cmd in ["cdo"]:
         run_and_check(f"{cmd} --help")
```

Comment:
Here and elsewhere, reminder to add
`module load cdo;` before running cdo
to prevent sys test error.

Reply:
Done with 164357e.